js/main.js

```diff
@@ -0,0 +1,9 @@
+// main.js
+// Contains Javascript functions.....
+
+// Easter egg
+var rawr = function() {

```

Comment:
Please 'hide' this function from the global scope so that I cannot invoke it directly from my browser's web console.

Reply:
Consider the module pattern (below)
https://medium.freecodecamp.org/javascript-modules-a-beginner-s-guide-783f7d7a5fcc#5be4